src/utils/Fetcher.ts

```diff
@@ -128,9 +169,14 @@ async function fetchInternal<T>(
           signal: controller.signal,
           body: options.body,
           method: options.method,
-          headers: options.headers
+          headers: getAllHeaders(options)
         })
         if (response.ok) {
+          if (setImmediateLambda) {
+            setImmediate(async () => {
```

Comment:
setImmediate does not work in browsers. I think you could use promise chining instead, with a similar practical effect

Reply:
I'm removing the `setImmediate` and just simply executing it.